terragrunt/aws/api/lambda.tf

```diff
@@ -0,0 +1,32 @@
+module "url_shortener_lambda" {
+  source                 = "github.com/cds-snc/terraform-modules?ref=v5.0.0//lambda"
+  name                   = "${var.product_name}-api"
+  billing_tag_value      = var.billing_code
+  ecr_arn                = aws_ecr_repository.api.arn
+  enable_lambda_insights = true
+  image_uri              = "${aws_ecr_repository.api.repository_url}:latest"
```

Comment:
Same comment here about using the variable instead of trying to access the resource directly.

It would also be good to add a variable for the tag so that in Staging we can always use `latest` and in Production, we can set a specific tag to have more control over the deployments.

Reply:
Added variables and I also added a new variable called ecr_tag to distinguish between versions.